doc/users/install.rst

```diff
@@ -16,7 +16,8 @@ image from Docker hub::
     docker pull nipype/nipype
 
 You may also build custom docker containers with specific versions of software
-using Neurodocker_ (see the :doc:`neurodocker`).
+using Neurodocker_ (see the `Neurodocker tutorial
+<https://miykael.github.io/nipype_tutorial/notebooks/introduction_neurodocker.html>`_).
```

Comment:
Can we move this to `nipype.github.io/tutorial`? Not sure if we already had discussion there...

Reply:
You're talking about the whole `nipype_tutorial`? Fine with me, but we haven't discussed it yet, or decided what to do.

Reply:
Yeah.